### PR TITLE
CU-9rt8wa Remove kapt

### PR DESCRIPTION
**Pull request by rachelcarmena**

### CODE REVIEW

**Comment by franciscodr** [comment]
Task linked: [CU-9rt8wa Remove Kotlin Annotation Processor Tool](https://app.clickup.com/t/9rt8wa)

**Review by nomisRev — APPROVED** [review]
Great thanks @rachelcarmena! 🎉

File: arrow-core-test/src/main/kotlin/arrow/core/Try.kt [hunk 1]
```diff
@@ -141,3 +141,10 @@ fun <A> A.success(): Try<A> = Success(this)
 fun Throwable.failure(): Try<Nothing> = Failure(this)
 
 fun <T> TryOf<TryOf<T>>.flatten(): Try<T> = fix().flatMap(::identity)
+
+class ForTry private constructor() { companion object }
```
Comment (aballano) [inline arrow-core-test/src/main/kotlin/arrow/core/Try.kt:145]: shouldn't Try be removed already for 0.12? 🤔

**Review by aballano — APPROVED** [review]
🔥 🔥

File: arrow-core-test/src/main/kotlin/arrow/core/Try.kt (hunk 1 above)
Comment (rachelcarmena) [reply arrow-core-test/src/main/kotlin/arrow/core/Try.kt:145]: It's still needed by kotlintest

Comment (aballano) [reply arrow-core-test/src/main/kotlin/arrow/core/Try.kt:145]: Soon then :D

**Pull request merged**